gitlab/macros.conf

```diff
@@ -10,10 +10,13 @@ iseval = 0
 definition = index=gitlab_webhook
 iseval = 0
 
+[gitlab_vulns]
+definition = | spath path=vulnerabilities{} output=vulnerability | mvexpand vulnerability | spath input=vulnerability | eval cve=if(source=="gitlab_cicd", mvfilter(match('identifiers{}.name', "^CVE")), source) | eval signature_id=if((source=="gitlab_cicd" AND isnotnull(description)), description, null()) | eval signature=if((source=="gitlab_cicd" AND isnotnull(description)), description, null()) | eval url=if((source=="gitlab_cicd" AND isnotnull('identifiers{}.url')), 'identifiers{}.url', null()) | eval xref=if((source=="gitlab_cicd" AND isnotnull('details.vulnerable_package.value')), 'details.vulnerable_package.value', null()) | eval "alert.number"=if((source=="gitlab_cicd" AND isnotnull(build_id)), build_id, null())
```

Comment:
Every time I see this it hurts my brain again like when I was originally trying to split this data up. Worth leaving in but I don't think we're directly leveraging it anymore. Is that correct to your knowledge?